phonebook/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"bufio"
+	"strings"
+)
+
+func  createPhoneBook(names []string, numbers []string) map[string]string{
+	m := make(map[string]string)
+	for i,key := range names {
+		m[key] = numbers[i]
+	}
+	return m 
+}
+	
+func main() {
+	name := []string 
+	numbers := []string
+	// я не понимаю в чем моя проблема в 17 строке 
+	f, err := os.Open("phone_numbers.txt")
+	bufio.NewScanner(a)
+	is name = true
+	for f.Scan() {
+		if is name {
+			names = append(names, s.Text())
+			is name = false
+		} else {
+			numbers = append(numbers, s.Text())
+			isname = true
+		}
+		}
+	}
```

Comment:
nit: This can also be replaced with 

```
for s.Scan() {
        names = append(names, s.Text())
        s.Scan()
        numbers = append(numbers, s.Text())
}
```